cmd/konvoy-image/cmd/aws.go

```diff
@@ -3,10 +3,9 @@
 package cmd
 
 import (
+	"github.com/mesosphere/konvoy-image-builder/pkg/app"
 	"github.com/spf13/cobra"
 	flag "github.com/spf13/pflag"
-
-	"github.com/mesosphere/konvoy-image-builder/pkg/app"
```

Comment:
This looks the gci config is wrong as i've linked above.

Reply:
got it. thanks